src/database/migrations/20190829010013-create-trip.js

```diff
@@ -0,0 +1,53 @@
+
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Trips', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      type: Sequelize.INTEGER,
+      primaryKey: true
+    },
+    tripId: {
+      type: Sequelize.STRING,
+      unique: true
+    },
+    userId: {
+      type: Sequelize.STRING,
+      references: {
+        model: 'Users',
+        key: 'userId'
+      }
+    },
+    departureDate: {
+      type: Sequelize.STRING,
```

Comment:
I feel this should be a type of Date and not String since that is what we are storing in it.

Reply:
I have fixed this